OpenTabletDriver.Desktop/Json/Converters/InterfaceConverter.cs

```diff
@@ -5,9 +5,11 @@ namespace OpenTabletDriver.Desktop.Json.Converters
 {
     public class InterfaceConverter<TInterface, TClass> : JsonConverter<TInterface> where TInterface : class where TClass : TInterface, new()
     {
+        public override bool CanWrite => false;
```

Comment:
to satisfy comment suggestion:
```suggestion
        // Newtonsoft.JSON custom `JsonConverter`s that only implement deserialization
        //      must specify CanWrite = false to use their default JSON writer
        // See upstream issue: https://github.com/JamesNK/Newtonsoft.Json/issues/2432
        public override bool CanWrite => false;
```